src/BuiltinExtensions/ComfyUIBackend/ExtraNodes/SwarmComfyExtra/SwarmYolo.py

```diff
@@ -40,6 +44,14 @@ def seg(self, image, model_name, index):
         else:
             masks = masks.data.cpu()
         masks = torch.nn.functional.interpolate(masks.unsqueeze(1), size=(image.shape[1], image.shape[2]), mode="bilinear").squeeze(1)
+
+        if sort_order != "default":
+            boxes = results[0].boxes
```

Comment:
most yolo models don't have boxes

Reply:
Again literally just use the existing sorting logic instead of building from scratch, it already accounts for different yolo model variants